### PR TITLE
Added Grow Button

### PR DESCRIPTION
**Pull request by parresh77**
Issue: #881

### CODE REVIEW

File: src/components/icon/_grow.less [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import "https://fonts.googleapis.com/css?family=Poppins:400,500,600,700&display=swap";
```
Comment (shahednasser) [inline src/components/icon/_grow.less:1]: Please remove this font as you are not using it.

File: src/components/icon/_grow.less [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.grow-btn {
+  display: inline-block;
+  font-family: "Arial";
```
Comment (shahednasser) [inline src/components/icon/_grow.less:4]: Remove this line

File: src/components/icon/_grow.less [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  overflow: hidden;
+
+  border-radius: 50px;
+  cursor: pointer;
+  box-shadow: 0px 10px 10px rgba(0, 0, 0, 0.1);
+  transition: all 0.3s ease-out;
+  &:hover {
+    &:after {
+      content: " Favourite";
+      font-family: "Arial";
```
Comment (shahednasser) [inline src/components/icon/_grow.less:18]: Remove this

File: src/components/icon/_grow.less [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      font-style: normal;
+      font-variant: normal;
+      text-rendering: auto;
+      -webkit-font-smoothing: antialiased;
+      transition: all 0.3s ease-out;
+    }
+    width: 200px;
+    margin: 0 0 1em 0;
+    transition: all 0.3s ease-out;
```
Comment (shahednasser) [inline src/components/icon/_grow.less:27]: remove ease-out

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for the great work! When I tested it the animation seemed a little laggy, so I think to make it better you should remove "ease-out" from the transition (I have added a comment below about it).
Also, I see you have used "Ariel" for the font-family. Instead please remove it and in `assets/css/index.css` add a rule for `.grow-btn` to use the font-family "Syne".

**Comment by parresh77** [comment]
Thanks for the review @shahednasser. I've made all the changes as you specified in the review.

File: assets/js/buttons-examples.js [hunk 5]
```diff
@@ -509,6 +509,15 @@ $(document).ready(function () {
           classes: "thumbsup-btn",
           text: "",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:512]: Place this in the correct alphabetical order under "Icon"

Comment (parresh77) [reply assets/js/buttons-examples.js]: Thanks for the feedback. I've placed it correctly now.

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @parresh77 for code

**Pull request merged**